src/signals/incident/definitions/wizard-step-2-vulaan/wonen.ts

```diff
@@ -778,6 +778,200 @@ const vakantieverhuur = {
   },
 }
 
+const verhuurderschap = {
+  extra_wonen_verhuurderschap_onderwerp: {
+    meta: {
+      ifOneOf: {
+        subcategory: 'goed-verhuurderschap',
+      },
+      label: 'Over welk onderwerp gaat uw melding?',
+      shortLabel: 'Onderwerp melding',
+      pathMerge: 'extra_properties',
+      values: {
+        discriminatie: 'Discriminatie',
+        intimidatie: 'Intimidatie',
+        huurcontract: 'Huurcontract',
+        bemiddelingskosten: 'Bemiddelingskosten',
+        arbeidsmigrant: 'Arbeidsmigrant',
+        servicekosten: 'Servicekosten',
+        overige: 'Overige (iets anders namelijk)',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.RadioInput,
+  },
+
+  // Discriminatie
+  extra_wonen_verhuurderschap_discriminatie: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label:
+        'Bent u gediscrimineerd tijdens het verhuurproces? We bedoelen in dit geval of u een woning niet heeft gekregen omdat u tot een specifieke groep behoort. Op welk grond bent u gediscrimineerd? Kies uit:',
+      shortLabel: 'Discriminatie groep',
+      pathMerge: 'extra_properties',
+      values: {
+        ras: 'Ras',
+        geloofsovertuiging: 'Geloofsovertuiging',
+        politieke_overtuiging: 'Politieke overtuiging',
+        gesalcht: 'Geslacht',
+        nationaliteit: 'Nationaliteit',
+        seksuele_gerichtheid: 'Seksuele gerichtheid',
+        burgelijke_staat: 'Burgerlijke staat',
+        handicap: 'Handicap',
+        chonische_ziekte: 'Chronische ziekte',
+        anders: 'Een andere reden, namelijk ',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.CheckboxInput,
+  },
+  extra_wonen_verhuurderschap_discriminatie_anders: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_discriminatie: 'anders',
+      },
+      label: 'Een andere reden, namelijk:',
+      shortLabel: 'Andere reden',
+      pathMerge: 'extra_properties',
+      type: 'url',
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.TextInput,
+  },
+  extra_wonen_verhuurderschap_discriminatie_bewijs: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label: 'Aanvullende informatie:',
+      shortLabel: 'Bewijs discriminatie',
+      pathMerge: 'extra_properties',
+      values: {
+        bewijs:
+          'Kunt u de discriminatie tijdens het verhuurproces aantonen door bijvoorbeeld een afwijzing van de woning of de selectiecriteria voor het verkrijgen van de woning? (Zo ja, voeg deze stukken toe aan de bijlage)',
```

Comment:
Niet dat jij hier iets aan kan doen, maar deze tekst hier is heel verwarrend. Je kan helemaal geen bijlages uploaden hier of later in het formulier. Je moet daarvoor terug naar step 1, bij de foto's toevoegen ervoor kiezen om niet een foto maar een pdf toe te voegen (met bijvoorbeeld de afwijzing van de woning). Hopelijk gaat men daar nog iets anders voor terug zetten.